Improve the UI

Michal-Frishman opened the pull request:
Can you please review it?
Can you try to locate the hamburger icon to be in the left side?
Meanwhile it is in the middle like the navigation links, I didn't succeed to correct it.

chayaziv commented:
its seems good 
but what about the separtion to sections like in the google sheet?
what about the react router?
all the navigates suposs to be in file named Router.jsx wuth function createreactrouter...!
no that?
about the UI i will check

chayaziv commented:
the UI of the heade is OK
but it will be more effective 
if you will fix all about the header and then PR + merge
and then all about the Home page and then PR  + merge....
when i say **all** - i mean all the rules in the google sheet
but its your choice , we dont must, maybe we give up on it ??

Michal-Frishman commented:
Of course it is better to finish all the rules before merging.
I will tell you when it has done.

Michal-Frishman commented:
I improved the structure of the component,
added a custom hook,
fixed the responsiveness,
and removed the sign up button (not needed for now).
I would appreciate it if you could go through the code to check that everything is good.

chayaziv commented:
yes, we make type only to the manily component
not to the functions
Now the code is completely ready
and you can merge it 
Vary nice!

chayaziv commented:
try with close
